tests/wpt/metadata/fetch/api/request/request-consume.html.ini

```diff
@@ -6,37 +6,7 @@
   [Consume String request's body as formData]
     expected: FAIL
 
-  [Consume Int8Array request's body as text]
-    expected: FAIL
-
-  [Consume Int8Array request's body as blob]
-    expected: FAIL
-
-  [Consume Int8Array request's body as arrayBuffer]
-    expected: FAIL
-
-  [Consume ArrayBuffer request's body as text]
-    expected: FAIL
-
-  [Consume Float32Array request's body as blob]
-    expected: FAIL
-
-  [Consume Float32Array request's body as arrayBuffer]
-    expected: FAIL
-
-  [Consume Float32Array request's body as JSON]
-    expected: FAIL
-
-  [Consume DataView request's body as text]
-    expected: FAIL
-
-  [Consume DataView request's body as blob]
-    expected: FAIL
-
-  [Consume DataView request's body as arrayBuffer]
-    expected: FAIL
-
-  [Consume DataView request's body as JSON]
+  [Trying to consume bad JSON text as JSON: 'undefined']
```

Comment:
I had a little doubt, whether it should be here ?

Reply:
The easiest way to find out is to run `./mach test-wpt --no-pause-after-test tests/wpt/fetch/api/request/request-consume.html` and see if any unexpected results are reported.